js/luno.js

```diff
@@ -126,6 +126,44 @@ module.exports = class luno extends Exchange {
         return this.parseOrderBook (orderbook, timestamp, 'bids', 'asks', 'price', 'volume');
     }
 
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        let response = await this.privateGetOrders (this.extend ({
+            'id': id.toString (),
+        }, params));
+        let state = (response.state == 'PENDING') ? 'open' : 'closed';
```

Comment:
I suggest, let's add some fixes to it on the way.

This particular line breaks the rule from https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

`response.state` will only work in JS, so we need to write `response['state']` and be very careful with formatting, because the library code is transpiled into 3 languages. We have to follow the rules very precisely, otherwise it won't build. So we always use square brackets everywhere, when we do assoc key-value access.